tests/bug-130/suricata.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
```

Comment:
I don't think it will be needed to provide a yaml in this case. The default yaml should also have the relevant output enabled